src/bootman/bootman.h

```diff
@@ -350,9 +352,35 @@ static inline void kernel_array_free(void *v)
         nc_array_free(&a, (array_free_func)free_kernel);
 }
 
+/**
+ * Get freestanding initrd
+ */
+bool boot_manager_get_initrd_freestanding(BootManager *self);
```

Comment:
This will be much better if called `boot_manager_enumerate_initrd_freestanding()`. It does not "get" anything, but rather initializes.